interfaces/vrep/DQ_VrepInterface.m

```diff
@@ -152,6 +172,25 @@ function disconnect_all(obj)
             %% Flushes all V-REP remote API connections from the server
             obj.vrep.simxFinish(-1);
         end
+
+        %% Set Synchronous Mode
+        function set_synchronous(obj,flag)
```

Comment:
Please, improve the documentation. I wonder if `flag` indicates whether the synchronous mode is enabled or disabled.